content/en/xlt/load-testing/Manual/320-test-evaluation.md

```diff
@@ -159,17 +159,19 @@ The Page Load Timings section offers deeper insight into the page loading perfor
 These timings will be recorded only when using `XltChromeDriver` or `XltFirefoxDriver` to run the browser. These are special WebDriver implementations that install an extension into the browser which is able to gather all the timings and report them to XLT.
 {{% /note %}}
 
-### Custom Timers & Values
+### Custom Timers and Custom Data
 
-The custom timers includes all timers that have been placed individually within the test code. The chart and data description is identical to the request section. In case custom samplers have been run during the test, the collected data is shown in the *Custom Values* section.
+The _Custom Timers_ page includes all [timers]({{< relref "../advanced/050-custom-data/#custom-timers" >}}) that have been placed individually within the test code. The chart and data description is identical to the _Actions_ section. 
+
+The _Custom Data_ page contains two sections: in case [custom samplers]({{< relref "../advanced/050-custom-data/#custom-values" >}}) have been run during the test, the collected data is shown in the *Custom Values* section. If the test scripts have also been collecting [custom data logs]({{< relref "../advanced/050-custom-data/#custom-data-logs" >}}), they will be available for download in the _Custom Data Logs_ section.
```

Comment:
```suggestion
The _Custom Data_ page contains two sections. In case [custom samplers]({{< relref "../advanced/050-custom-data/#custom-values" >}}) have been run during the test, or if your test scenarios are instrumented to log custom values, the collected data is shown in the *Custom Values* section. If the test scripts have also been collecting [custom data logs]({{< relref "../advanced/050-custom-data/#custom-data-logs" >}}), they will be available for download in the _Custom Data Logs_ section.
```

Reply:
For some reason, this change was undone when splitting the section into two section.

Reply:
Fixed.